packages/rpc-subscriptions-spec/README.md

```diff
@@ -14,4 +14,44 @@
 
 # @solana/rpc-subscriptions-spec
 
-TODO
+This package contains types that describe the implementation of the JSON RPC Subscriptions API, as well as methods to create one. It can be used standalone, but it is also exported as part of the Solana JavaScript SDK [`@solana/web3.js@rc`](https://github.com/solana-labs/solana-web3.js/tree/master/packages/library).
+
+This API is designed to be used as follows:
+
+```ts
+const rpcSubscriptions =
+    // Step 1 - Create an `RpcSubscriptions` instance. This may be stateful.
+    createSolanaRpcSubscriptions(mainnet('wss://api.mainnet-beta.solana.com'));
+const response = await rpcSubscriptions
+    // Step 2 - Call supported methods on it to produce `PendingRpcSubscriptionsRequest` objects.
+    .slotNotifications({ commitment: 'confirmed' })
+    // Step 3 - Call the `subscribe()` method on those pending requests to trigger them.
+    .subscribe({ abortSignal: AbortSignal.timeout(10_000) });
+// Step 4 - Iterate over the result.
+try {
+    for await (const slotNotification of slotNotifications) {
+        console.log('Got a slot notification', slotNotification);
+    }
+} catch (e) {
+    console.error('The subscription closed unexpectedly', e);
+} finally {
+    console.log('We have stopped listening for notifications');
+}
+```
```

Comment:
I think this example is more suited for `rpc-subscriptions` than `rpc-subscriptions-spec` since it's the highest-level API we offer.

Perhaps it would be better to offer examples on how to use the new type here instead?

Reply:
I literally just copy-pasted this from `@solana/rpc-spec`.

I honestly don't understand the differences anymore.

Help me, Obi Wan…